### PR TITLE
WIP issue #34: Update landing page

### PR DESCRIPTION
**Pull request by em-henken**
starting to recreate UI for landing page, site Nav, etc.

### CODE REVIEW

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
Keep the button after the text, as the heading and button have same text, its confusing to the user

**Review by yashaswini-slu — CHANGES_REQUESTED** [review]
Can you add some more look and feel for the page, like some border both the option, something like this:
Screenshot 2023-10-21 at 19 18 50
This is just an idea, you can think over it

File: public/index.html [hunk 1]
```diff
@@ -21,42 +21,33 @@
             <div class="col">
                 <h3> Welcome to the RERUM Geolocator </h3>
                 <p class="intro">
-                    Here you can enhance any <a target="_blank" href="https://www.w3.org/standards/semanticweb/data.html">Linked Open Data</a> resource by providing geospatial descriptors. This does not affect the original resource nor does it notify the resource owner.  There is no user system and so data generated here cannot be attributed to you specifically.  However, it will be available to you to make it your own by taking it through applications of your choosing.
+                    Here you can enhance any <a target="_blank" href="https://www.w3.org/standards/semanticweb/data.html">Linked Open Data</a> resource by providing geospatial descriptors. This does not affect the original resource nor does it notify the resource owner. You are able to attribute your data to yourself, as well as download your object as a JSON file at the end.
```
Comment (thehabes) [inline public/index.html:24]: Sneak in the plug for RERUM here.

> You are able to attribute the data created here to yourself.  At the end, you can download your object as a JSON file or save it to RERUM as real usable linked data.

File: public/index.html [hunk 2]
```diff
@@ -21,42 +21,33 @@
...
-                </p>
+                    The navPlace property is for web resources that are IIIF Presentation API Defined Types. <a target="_blank" href="https://iiif.io/api/presentation/3.0/#21-defined-types".> </a> Learn more about IIIF and the navPlace property by exploring <a target="_blank" href="https://iiif.io/api/extension/navplace/">navPlace Extension</a>.
+                <br></br>
```
Comment (thehabes) [inline public/index.html:34]: If you can avoid it, don't use `<br>`

File: public/index.html [hunk 3]
```diff
@@ -21,42 +21,33 @@
...
-                <h4> <abbr title="International Image Interoperability Framework">IIIF</abbr> Resources </h4>
-                <p>
-                    This application is proud to support the <a target="_blank" href="https://iiif.io/api/presentation/">IIIF Presentation API</a> and encourages you to bring your own IIIF resources.  If your web resource fits this category, there are two main paths to take.
-                </p>
-                <h5> navPlace </h5>
+
+        <div class="row" >
+            <div class="col text-center" style="padding-right:20px; padding-left:10px; background: radial-gradient(circle, #bdd5bd, #e4ece4); border-radius: 15px;">
+                <h4> Create a navPlace object </h4>
                 <p>
```
Comment (thehabes) [inline public/index.html:32]: This `<p>` doesn't close!  When you close it, you can give it a padding or margin to get the spacing effect you are going for (this is in relation to the `br` comment).

File: public/index.html [hunk 4]
```diff
@@ -21,42 +21,33 @@
...
-        </div>
-        <div class="row">
-            <div class="col">
-                <h4> Other Linked Open Data Resources </h4>
+        
+        
+            <div class="col text-center" style="padding-left:20px; padding-right:10px; background: radial-gradient(circle, #bdd5bd, #e4ece4); border-radius: 15px;">
+                <h4> Create a Geolocating Web Annotation </h4>
+                
                 <p>
```
Comment (thehabes) [inline public/index.html:44]: This <p> doesn't close! When you close it, you can give it a padding or margin to get the spacing effect you are going for (this is in relation to the br comment).

File: public/index.html [hunk 5]
```diff
@@ -21,42 +21,33 @@
...
-                    <a target="_blank" href="https://lod-cloud.net/">Linked Open Usable Data is a vast expanse</a> of different data classifications with well described taxonomies and ontologies.  If your URI resolves to JSON, preferably JSON-LD <abbr title="?"><a target="_blank" href="https://dashjoin.medium.com/json-schema-schema-org-json-ld-whats-the-difference-e30d7315686a">(what's the difference?)</a></abbr>, then you can provide GeoJSON-LD for that <abbr title="Linked Open Usable Data">LOUD</abbr> entity through using a <a target="_blank" href="https://www.w3.org/TR/annotation-model/">W3C Web Annotation</a>.  This path seeks to finish with a real active <abbr title="Linked Open Usable Data">LOUD</abbr> Geolocating Web Annotation for the resource you provide.
-                </p>
-                <h5>W3C Web Annotation</h5>
+                    Web Annotations are for any web resource.  To explore what exactly this means, learn about <a target="_blank" href="https://www.w3.org/TR/annotation-model/">W3C Web Annotation</a> and <a target="_blank" href="https://lod-cloud.net/">Linked Open Usable Data</a>.
+                <br></br>
```
Comment (thehabes) [inline public/index.html:46]: If you can avoid it, don't use <br>

**Review by thehabes — CHANGES_REQUESTED** [review]

**Review by thehabes — APPROVED** [review]
There is inline CSS all over the place here.  CSS should be applied via /stylesheets/style.css or by using existing Chota classes.  Either class your objects and create new CSS rules or take over the existing CSS rules.  It's OK for today, you can make this change during your next sprint.  I understand some of the inline CSS was already there.  Someone should have yelled at the developer that did that.

**Review by yashaswini-slu — APPROVED** [review]

**Pull request merged**